debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/wal2json/StreamingWal2JsonMessageDecoder.java

```diff
@@ -197,8 +194,7 @@ protected void outOfOrderChunk(final byte[] content) {
             LOGGER.warn("Got out of order chunk {}, recording artifical TX", new String(content));
         }
         txId = UNDEFINED_LONG;
-        timestamp = UNDEFINED_STRING;
-        commitTime = null;
+        commitTime = Instant.now();
```

Comment:
@jpechane, could you elaborate a bit on that entire notion of "out of order chunk"? E.g. in form of a method comment. Setting the commit time to "now" doesn't reflect reality AFAICS, should we log a warning in that method perhaps? Or is it something which will happen regularly?

Reply:
@gunnarmorling This happens sometimes when the database is restarted after the crash, unfrotunately not reproducible.

Reply:
Ok, can you add a method comment?

Reply:
@gunnarmorling javaDoc comment added

Reply:
Thx.